### ISSUE TITLE
No message when removing choices from a multiselect

### ISSUE THREAD

**Issue by bakera3** [CompositionalIT/Feliz.ReactSelect#5]
I am attempting to use onChange to handle messages of both adding choices in a multiselect as well as removing them.  After adding multiple choices, there is no message when I delete an individual choice nor when I use the isClearable button the delete all choices from the dropdown.

```fsharp
            ReactSelect.create [
                ReactSelect.isSearchable true
                ReactSelect.isMulti true
                ReactSelect.isClearable false
                ReactSelect.options (
                    getDropdownValues dmsedata columnMapping
                    |> Array.map(fun dropdown ->
                        {
                            Label = dropdown
                            Value = dropdown
                        }
                        )
                    )
                ReactSelect.menuPlacement Bottom
                ReactSelect.onChange (
                    fun (choices: Choice []) ->
                    choices
                    |> Array.iter(fun choice ->
                        dispatch (AddFilter (fst columnMapping, choice?label))
                        )
                    )
                ]
```

**Comment by Tom-Sloboda** [comment]
We tried to reproduce the issue by replacing `onChange` code you have with `ReactSelect.onChange (fun (choices: Choice []) -> Browser.Dom.console.log $"Currently selected: {choices}")` and it seems to print the selected choices correctly on every change.

Could you please provide a minimal example that reproduces the issue?

**Comment by bakera3** [comment]
Hello Tom,

I spent some time testing your tweak above and it does appear to be functioning correctly now.  There are two possibilities that come to mind as the cause here.  

1. I had to pin some dependency versions to address an issue in feliz-ag-grid which might have had other consequences. 
2. It is possible I was poorly handling the choice array messaging coming out of onChange and I just happened to fix it without meaning to.

Either way, thank you for your response!
